src/pages/[platform]/build-a-backend/auth/manage-users/manage-devices/index.mdx

```diff
@@ -326,6 +326,26 @@ Future<void> fetchAllDevices() async {
 ```
 
 </InlineFilter>
+
+## Fetch the current device
```

Comment:
It looks like these two lines should be inside the `InlineFilter` for flutter.  Currently they are outside the filter and will appear for every platform even though not every platform has instructions for fetching the current device.